### PR TITLE
[doc] add general-relation grafana ui doc.

### PR DESCRIPTION
**Pull request by yswdqz**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

File: docs/en/setup/backend/ui-grafana.md [hunk 1]
```diff
@@ -101,6 +101,14 @@ to prevent it value be escaped.
 4. Test query and save the panel.
 <img src="https://skywalking.apache.org/screenshots/9.6.0/logql/grafana-logs-panel.jpg"/>
 
+## Configure General Relation DashBoard
```
Comment (wu-sheng) [inline docs/en/setup/backend/ui-grafana.md:104]:
```suggestion
## Configure General Relation Dashboards
```

File: docs/en/setup/backend/ui-grafana.md [hunk 2]
```diff
@@ -101,6 +101,14 @@ to prevent it value be escaped.
 4. Test query and save the panel.
 <img src="https://skywalking.apache.org/screenshots/9.6.0/logql/grafana-logs-panel.jpg"/>
 
+## Configure General Relation Dashboards
+The following steps are the example of config a `General Relation` dashboard:
+1. Create a dashboard named `General Relation`.
+2. Configure variables for the dashboard:
+   <img src="https://skywalking.apache.org/screenshots/10.0.1/promql/grafana-relation-variables1.jpg"/>
+   After configure, you can select the service/instance/endpoint and the dest service/instance/endpoint on the top of the dashboard:
+   <img src="https://skywalking.apache.org/screenshots/10.0.1/promql/grafana-relation-variables2.jpg"/>
```
Comment (wu-sheng) [inline docs/en/setup/backend/ui-grafana.md:110]: We need to wait for https://github.com/apache/skywalking-website/pull/722

File: docs/en/setup/backend/ui-grafana.md [hunk 3]
```diff
@@ -101,6 +101,14 @@ to prevent it value be escaped.
 4. Test query and save the panel.
 <img src="https://skywalking.apache.org/screenshots/9.6.0/logql/grafana-logs-panel.jpg"/>
 
+## Configure General Relation Dashboards
+The following steps are the example of config a `General Relation` dashboard:
+1. Create a dashboard named `General Relation`.
+2. Configure variables for the dashboard:
+   <img src="https://skywalking.apache.org/screenshots/10.1.0/promql/grafana-relation-variables1.jpg"/>
+   After configure, you can select the service/instance/endpoint and the dest service/instance/endpoint on the top of the dashboard:
+   <img src="https://skywalking.apache.org/screenshots/10.1.0/promql/grafana-relation-variables2.jpg"/>
```
Comment (wankai123) [inline docs/en/setup/backend/ui-grafana.md:110]: I think you should add an example dashboard config screenshot with the config step. You can refer the other section above.

Comment (wankai123) [reply docs/en/setup/backend/ui-grafana.md:110]: here is only variables

Comment (yswdqz) [reply docs/en/setup/backend/ui-grafana.md:110]: Got it, But I think the rest is almost the same as the general dashboard, how should I phrase it (just write 'same as general' or add a new part)?

Comment (wankai123) [reply docs/en/setup/backend/ui-grafana.md:110]: The PrpmQL labels are different, we need to tell users how to config a relation-metrics PromQL

File: docs/en/setup/backend/ui-grafana.md [hunk 4]
```diff
@@ -101,6 +101,23 @@ to prevent it value be escaped.
 4. Test query and save the panel.
 <img src="https://skywalking.apache.org/screenshots/9.6.0/logql/grafana-logs-panel.jpg"/>
 
+## Configure General Relation Dashboards
```
Comment (wankai123) [inline docs/en/setup/backend/ui-grafana.md:104]:
```suggestion
## Configure Relation Dashboards
```

**Review by wankai123 — APPROVED** [review]

**Pull request merged**